planet/api/dispatch.py

```diff
@@ -92,6 +93,10 @@ def rebuild_auth(self, prepared_request, response):
                     )
 
 
+def _get_user_agent():
+    return 'planet-python-client/%s' % __version__
```

Comment:
to reduce potential confusion, can we just use the project name?

Reply:
oh man i must have been dreaming and thought thats what the project name was...

Reply:
fixed